scripts/scil_mrds_metrics.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to compute all of the Multi-Resolution Discrete-Search (MRDS) metrics.
```

Comment:
metrics -> multi-tensor metrics

Reply:
Now that I have seen the code, I'm not sure what this script does... single-tensor metrics only?